jwql/website/apps/jwql/oauth.py

```diff
@@ -162,8 +164,11 @@ def user_info(request, **kwargs):
 
         # If user is authenticated, return user credentials
         if cookie is not None:
+            check_config('auth_mast')
+            auth_mast = get_config()['auth_mast']
+
             response = requests.get(
```

Comment:
Just as a sidenote: I tried this with the MAST ops server (specified via `auth_mast`) and it failed, probably because JWQL is not registered there yet.